src/plugins/vis_types/timeseries/public/application/components/lib/check_if_series_have_same_formatters.ts

```diff
@@ -1,33 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0 and the Server Side Public License, v 1; you may not use this file except
- * in compliance with, at your election, the Elastic License 2.0 or the Server
- * Side Public License, v 1.
- */
-
-import { last, isEqual } from 'lodash';
-import { DATA_FORMATTERS } from '../../../../common/enums';
-import type { Series } from '../../../../common/types';
-import type { FieldFormatMap } from '../../../../../../data/common';
-
-export const checkIfSeriesHaveSameFormatters = (
```

Comment:
This logic doesn't have a lot of sense. We need to rely on the first formatter in the group like Lens does.